ui/src/rdf-vocabularies.ts

```diff
@@ -0,0 +1,44 @@
+import rdf from 'rdf-ext'
+import { Quad } from 'rdf-js'
+import { vocabularies, prefixes, expand as rdfExpand, shrink as rdfShrink } from '@zazuko/rdf-vocabularies'
+
+const relevantPrefixes = ['rdfs', 'schema', 'qb', 'sdmx', 'dcterms', 'dc11', 'skos', 'skosxl', 'xkos', 'xsd', 'wgs']
+
+const PROPERTY = rdf.namedNode(rdfExpand('rdf:Property'))
+const TYPE = rdf.namedNode(rdfExpand('rdf:type'))
```

Comment:
A simpler way is to use `@rdjfs/namespace` to create URI terms.

```
import { namespace } from '@rdfjs/namespace'
import { prefixes } from '@zazuko/rdf-vocabularies'

const rdf = namespace(prefixes.rdf)

const TYPE = rdf.type
```

Reply:
This package has 14 lines of code... I'll never understand the JS ecosystem :(

But I guess I can get rid of rdf-ext

Reply:
I replaced "rdf-ext" with "@rdfjs/data-model". Hopefully it's a lighter dependency?

Reply:
https://bundlephobia.com/ is a good service for this: @rdfjs/data-model is 3% the size of rdf-ext :)

Reply:
Nice! :)

Reply:
Alcaeus has a dependency on rdf-ext anyway. But I should remove that too when I finish 1.0 (not long from now)